docs/en/solutions/apply-catboost-model.md

```diff
@@ -0,0 +1,212 @@
+# Applying a CatBoost model in ClickHouse {#applying-catboost-model-in-clickhouse}
+
+[CatBoost](https://catboost.ai) — is a free and open-source gradient boosting library for machine learning.
+
+To apply a CatBoost model in ClickHouse:
+
+1. [Create a table for the train sample](#create-a-table).
```

Comment:
why it's called "train sample"? we're just running inference here, not training a model (btw adding a link to docs on how to train CatBoost models would be useful)